app.js

```diff
@@ -4,11 +4,12 @@ var ari = require('ari-client');
 var util = require('util');
 var sla = require('./lib/sla.js');
 var Q = require('q');
+var dal = require('./lib/dal.js');
 
 var connect = Q.denodeify(ari.connect);
 var confFile;
 if (confFile = process.argv[2]) {
-  connect('http://127.0.0.1:8088', 'user', 'pass')
+  connect('http://localhost:8088', 'user', 'pass')
```

Comment:
I'd revert this change, and/or do it in a separate pull request.

Your pull requests should focus on solving the problem at hand: in this case, implementing call hold. Ancillary changes, such as switching from a specific IPv4 representation of a local loopback to a hostname resolution for a local machine is ancillary to your task, and should be done as a separate commit/pull request.